.openshift-ci/jobs/integration-tests/env.sh

```diff
@@ -33,3 +38,18 @@ shopt -s nullglob
 for cred in /tmp/secret/**/[A-Z]*; do
     export "$(basename "$cred")"="$(cat "$cred")"
 done
+
+# only run all integration tests on master, or when the all-integration-tests
+# label is added. This is checked in this common env script because it allows
+# us to skip pre- and post- steps as well (which source this file)
+if ! pr_has_label "all-integration-tests" && is_in_PR_context; then
```

Comment:
[ultra-nit] Maybe we should check if we are in a PR before attempting to gather the labels?
```suggestion
if is_in_PR_context && ! pr_has_label "all-integration-tests"; then
```